src/global/privileges_drop.cc

```diff
@@ -1,6 +1,9 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
+#ifdef WITH_SEASTAR
```

Comment:
@rzarzynski just a nit, might want to fix the commit message:

> thatbrakes

Reply:
Fixed!